frontend/packages/kubevirt-plugin/integration-tests-cypress/views/dialog.ts

```diff
@@ -68,5 +94,20 @@ export const addDisk = (disk: Disk) => {
   if (disk.preallocation) {
     cy.contains('Enable preallocation').click();
   }
+  if (disk.autoDetach === true) {
+    cy.get(diskDialog.autoDetach)
+      .check()
+      .should('be.checked');
+  } else if (disk.autoDetach === false) {
+    cy.get(diskDialog.autoDetach)
+      .uncheck()
+      .should('not.be.checked');
+  }
   cy.get(diskDialog.add).click();
 };
+
+export const delDisk = (name: string) => {
```

Comment:
Would you mind renaming this to `deleteDisk`? It's short enough when spelled out that I don't think it needs to be shortened.

Reply:
Sure, will do.